src/carplay.cs

```diff
@@ -383,6 +595,7 @@ interface ICPListTemplateDelegate { }
 	interface CPListTemplateDelegate {
 
 		[Abstract]
+		[Async]
```

Comment:
From the docs, I don't think this is an async call: 

> When implementing this method, perform any necessary operations to prepare for completing the item selection, including updating your user interface. You must call completionHandler after handling the selected item to tell the system that it can continue.


Reference: https://developer.apple.com/documentation/carplay/cplisttemplatedelegate/2977578-listtemplate?language=objc